docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -1,41 +1,53 @@
-
-.. _intro_to_sawtooth:
-
-*****************************************
-Getting Started with Hyperledger Sawtooth
-*****************************************
-
+****************************
+Installation on Ubuntu 16.04
+****************************
 
 Overview
 ========
 
-This tutorial shows you how to start a validator and transaction processor,
-then perform some basic operations, such as:
+This tutorial guides you through the process of setting up Hyperledger Sawtooth
+for application development on Ubuntu Linux, and then introduces basic Sawtooth
+concepts necessary for application development. After this tutorial, you will be
+ready to perform app development tasks, such as implementing business logic with
+transaction families and writing clients which use Sawtooth's REST API.
 
-* Submitting transactions
-* Setting configuration settings using the config transaction family
-* Viewing blocks, transactions and state
+If you want to install Sawtooth using Docker Compose rather than Docker Compose, see :doc:`docker`. 
 
-If you have not yet set your environment up, see :doc:`environment_setup` in
-the App Developer's Guide.
 
 Prerequisites
--------------
+=============
+
+Sawtooth requires Ubuntu version 16.04 or higher.
```

Comment:
Remove this section.  The overall content is for "Using Sawtooth on Ubuntu 16.04".  We do not support any other Ubuntu version with our current packaging.  Sawtooth itself might work though, if installed via some other method, so "Sawtooth requires" is not accurate.